image.go

```diff
@@ -514,7 +515,7 @@ func (i *Image) DrawTriangles(vertices []Vertex, indices []uint16, img *Image, o
 			vs[i*graphics.VertexFloatCount+7] = v.ColorA * ca
 		}
 	}
-	is := make([]uint32, len(indices))
+	is := i.ensureTmpIndices(len(indices))
```

Comment:
We can do the same thing at `DrawTrianglesShader`

Reply:
done